pyproject.toml

```diff
@@ -25,6 +25,7 @@ sqlalchemy = "^2.0.31"
 langchain-community = "^0.2.12"
 neo4j = "^5.23.1"
 rank-bm25 = "^0.2.2"
+pymilvus = "^2.4.6"
```

Comment:
We don't want to include this as core dependency. Can you prompt the user if Milvus is not installed when they try to use `MilvusDB`